pkg/install/app/vaultengineconfigdeps.go

```diff
@@ -0,0 +1,235 @@
+package app
+
+import (
+	"context"
+	"strings"
+
+	batchv1obj "github.com/puppetlabs/leg/k8sutil/pkg/controller/obj/api/batchv1"
+	corev1obj "github.com/puppetlabs/leg/k8sutil/pkg/controller/obj/api/corev1"
+	"github.com/puppetlabs/leg/k8sutil/pkg/controller/obj/helper"
+	"github.com/puppetlabs/leg/k8sutil/pkg/controller/obj/lifecycle"
+	"github.com/puppetlabs/relay-core/pkg/apis/install.relay.sh/v1alpha1"
+	"github.com/puppetlabs/relay-core/pkg/model"
+	"github.com/puppetlabs/relay-core/pkg/obj"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	vaultAddr                     = "http://vault:8200"
```

Comment:
Might be worth a comment here that this corresponds exactly to the default value for the external address (unless I'm misunderstanding how this all gets passed around), so if you update one, you have to update the other for this code to continue to behave as expected.